### DISCUSSION TITLE
GeoCAT-examples & GeoCAT-viz 2023 roadmap

### DISCUSSION THREAD

**Discussion by erogluorhan** [NCAR/GeoCAT-examples#492] [Announcements]
This discussion can serve as: 

1. Announcement of what we are planning to do overall in the 2023 calendar year for GeoCAT-examples & GeoCAT-viz, as part of the [GeoCAT 2023 roadmap](https://geocat.ucar.edu/pages/roadmap.html)
2. Feedback/input point for the community on our roadmap

**GeoCAT-examples & GeoCAT-viz 2023 roadmap (from [GeoCAT 2023 roadmap](https://geocat.ucar.edu/pages/roadmap.html)):**

1. Development
    - New functionality
       - New examples from community requests
       - New examples from ordered list (TBD)
       - Changes to existing examples for clarity
    - Improvement, maintenance, and quality of life updates
       - Support for mesh-based rendering
       - Interactive plots, beginning with support for time step animations
       - Changes to existing examples to keep up with dependencies
       
2. Open development & community engagement
    - Assisting SIParCS internships for Virginia Do
    - Provide content and presentations for educational content throughout the year

**Comment by bitterbark** [comment]
Is this still the right place to make community requests? I'm part of the ADF and MDTF teams and was working with a GeoCAT member who subsequently left to convert [NCL's implementation of MJO analysis functions](https://www.ncl.ucar.edu/Applications/mjoclivar.shtml) to python in order to update our existing code. I'd love to restart this collaboration if it fits in your priorites.

**Reply by kafitzgerald** [reply]
Thanks for raising this again as a feature request!  

I see the [old issue](https://github.com/NCAR/geocat-comp/issues/30) over on geocat-comp and will bump the priority on that.  That's probably the best place to continue chatting about this, but you should be able to get our attention if you open and issue or discussion on most GeoCAT projects.  There's also this [feature request form](https://docs.google.com/forms/d/e/1FAIpQLSeWuogl8Z671et3DdrmglenBPKGFmHVHkpSamTQeoPGuytLZw/viewform) if that's easier.  

Is the MJO suite in the MDTF diagnostics still the place to look for use cases you're looking to port to Python?

**Reply by bitterbark** [reply]
Absolutely! 
Thanks for finding the old issue, I had lost it! I will continue discussion there.
And thanks for the quick response.
